apiservers/engine/backends/container.go

```diff
@@ -37,18 +40,19 @@ import (
 )
 
 type Container struct {
+	m sync.Mutex
```

Comment:
This is nitpicky, but you have a mutex in this struct protecting a data structure out of the struct.  Since it's a temporary solution, it's not a big deal, but it would be nice to move the map in the struct to be within the same space as the mutex.